test/client/js/libs/handlebars/modesTest.js

```diff
@@ -0,0 +1,18 @@
+"use strict";
+
+const expect = require("chai").expect;
+const modes = require("../../../../../client/js/libs/handlebars/modes");
+
+describe("modes Handlebars helper", function() {
+	it("should return a text mode based on symbol", function() {
+		expect(modes("@")).to.equal("op");
```

Comment:
Test all the known modes.

Reply:
Will do. I wasn't sure if that was a meaningful test to do.